bears/general/CPDBear.py

```diff
@@ -0,0 +1,41 @@
+import re
+import os
+
+from coalib.bearlib.abstractions.Lint import Lint
+from coalib.bears.LocalBear import LocalBear
+
+
+class CPDBear(LocalBear, Lint):
```

Comment:
@aachiritoaei I'd rather make this global indeed before merging shouldn't be too hard should it?